examples/flax/language-modeling/run_t5_mlm_flax.py

```diff
@@ -290,7 +290,7 @@ def create_sentinel_ids(self, mask_indices):
         start_indices[:, 0] = mask_indices[:, 0]
 
         sentinel_ids = np.where(start_indices != 0, np.cumsum(start_indices, axis=-1), start_indices)
```

Comment:
Is that line not completely unecessary since we have the line right after?

Reply:
Well the next line makes use of the just changed `sentinel_ids` parrameter no?

Reply:
Sorry what I meant is those two lines should be summarizable to 
```python
sentinel_ids = np.where(start_indices != 0, (len(self.tokenizer) - np.cumsum(start_indices, axis=-1)), 0)
```
Ie what is 0 is kept at 0 and what's not 0 is given a non 0 value, which means that the next `where` operation uses the same segmentation and thus overrides the values.

cc @patil-suraj